test/foundry/FulfillOrderTest.t.sol

```diff
@@ -510,6 +510,48 @@ contract FulfillOrderTest is BaseOrderTest {
         );
     }
 
+    function testFulfillOrderRevertInvalidConsiderationItemsLength(
+        uint256 fuzzTotalConsiderationItems,
+        uint256 fuzzAmountToSubtractFromConsiderationItemsLength
+    ) public {
+        uint256 totalConsiderationItems = fuzzTotalConsiderationItems % 200;
+        // Set amount to subtract from consideration item length
+        // to be at most totalConsiderationItems.
+        uint256 amountToSubtractFromConsiderationItemsLength = totalConsiderationItems >
+                0
+                ? fuzzAmountToSubtractFromConsiderationItemsLength %
+                    totalConsiderationItems
+                : 0;
+
+        // Create order
+        (
+            Order memory _order,
+            OrderParameters memory _orderParameters,
+
+        ) = _prepareOrder(1, totalConsiderationItems);
+
+        // Get the calldata that will be passed into fulfillOrder.
+        bytes memory fulfillOrderCalldata = abi.encodeWithSelector(
+            consideration.fulfillOrder.selector,
+            _order,
+            conduitKeyOne
+        );
+
+        _performTestFulfillOrderRevertInvalidArrayLength(
+            consideration,
+            _order,
+            fulfillOrderCalldata,
+            // Order parameters starts at 0xa4 relative to the start of the
+            // order calldata because the order calldata starts with 0x20 bytes
+            // for order calldata length, 0x04 bytes for selector, and 0x80
+            // bytes until the start of order parameters.
+            0xa4,
```

Comment:
Would be good to clarify the 0x80 further but I don't have a good explanation, similar to above https://github.com/ProjectOpenSea/seaport/pull/461/files#r893103417